mobile/android/chrome/browser.properties

```diff
@@ -35,19 +37,32 @@ alertSearchEngineDuplicateToast='%S' już jest na liście wyszukiwarek
 downloadCancelPromptTitle=Anuluj pobieranie
 downloadCancelPromptMessage=Czy na pewno anulować pobieranie tego pliku?
 
-# LOCALIZATION NOTE (addonError-1, addonError-2, addonError-3, addonError-4):
+addonError.titleError=Błąd
+addonError.titleBlocked=Zablokowano dodatek
+addonError.learnMore=Więcej informacji
+
+# LOCALIZATION NOTE (unsignedAddonsDisabled.title, unsignedAddonsDisabled.message):
+# These strings will appear in a dialog when Firefox detects that installed add-ons cannot be verified.
```

Comment:
Nit (only once for all occurrences): don't add LOCALIZATION NOTE comments - it is better to check them in en-US files right above original entities than possibly outdated and needlessly taking space in translated file.